easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.4.4-intel-2017a.eb

```diff
@@ -15,13 +14,16 @@ description = """A wrapper tool around Cutadapt and FastQC to consistently apply
 
 toolchain = {'name': 'intel', 'version': '2017a'}
 
-sources = ['%(namelower)s_v%(version)s.zip']
 source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
+sources = ['%(namelower)s_v%(version)s.zip']
+checksums = ['ccd7eecc73327da29230f3e7c7311a4e21b2cfe7282be18fa059ba900fc33116']
 
 install_cmd = "cp -a * %(installdir)s && chmod +x %(installdir)s/%(namelower)s"
 
+postinstallcmds = ["sed -i '1 i#!/usr/bin/env perl' %(installdir)s/%(namelower)s"]
```

Comment:
why not as a part of `install_cmd`?

Reply:
why not as a part of `postinstallcmds`?

Reply:
It's just a bit weird to use `postinstallcmds` if you're already using `install_cmd`...

`postinstallcmds` is intended for running commands that must be executed after the installation and can't be specified easily otherwise (e.g. to avoid hacking `installopts` with `" && <command1> && <command2>"`

Just a suggestion though, both approaches are actually fine I guess...